system-addon/content-src/components/Topics/_Topics.scss

```diff
@@ -63,4 +63,10 @@
       transform: scaleX(-1);
     }
   }
+
+  &::after {
+    content: "";
+    display: table;
+    clear: both;
```

Comment:
Let's have a comment describing why this is needed. For both this clearfix and the original float.